### DISCUSSION TITLE
encounter issues during the re-verification process

### DISCUSSION THREAD

**Discussion by arijitbarai** [community/community#139135] [GitHub Education]
### Hi everyone,

Hey everyone,

I’m having some trouble reissuing my GitHub Education discount, and I was hoping someone here might have experienced something similar and could help out.

My discount expired last month, and when I tried to reapply, I ran into a roadblock. The last time I applied, I didn’t need to use my school-issued email to verify my student status. Now, it seems the system won’t let me proceed without it, which is a problem since my school doesn't provide school-issued email to the students.

**Comment by alx-mp4** [comment]
Hi @arijitbarai , I recommend opening a ticket with GitHub support for this issue.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
